Library/Homebrew/cmd/tap.rb

```diff
@@ -60,7 +60,7 @@ def tap
         tap.fix_remote_configuration
       end
     elsif args.no_named?
-      puts Tap.names
+      puts Tap.select(&:installed?)
```

Comment:
```suggestion
      puts Tap.select(&:installed?).map(&:name).sort
```

To be backwards compatible.

Reply:
`Tap::each` is already sorted, and `Tap#to_s` is equal to `Tap#name`, so it already is backwards compatible.